Library/Homebrew/formula.rb

```diff
@@ -1810,6 +1818,17 @@ def mirror(val)
       define_method(type) { |val| stable.send(type, val) }
     end
 
+    # @!attribute [w] gpg
+    # An optional mechanism to verify the download with GnuPG as well as the
+    # standard checksum verification. Upstream must publish signature files
+    # for us to be able to use this mechanism. Those signature files often
+    # have either a ".gpg" or ".sig" file extension.
+    #
+    # <pre>gpg "https://ftpmirror.gnu.org/wget/wget-1.18.tar.xz.sig"</pre>
+    def gpg(val)
+      stable.gpg(val)
+    end
```

Comment:
> A checksum is pretty redundant, IMO.

I made a mistake here. You are right.

The real problem is I'm not sure how do you indent to manage the public keys? Because the most comment attack is replacing the source the signature at the same time